runtime/core/decoder/torch_asr_decoder.cc

```diff
@@ -120,8 +124,8 @@ bool TorchAsrDecoder::AdvanceDecoding() {
                                       ->run_method("ctc_activation", chunk_out)
                                       .toTensor()[0];
     encoder_outs_.push_back(std::move(chunk_out));
-    ctc_prefix_beam_searcher_->Search(ctc_log_probs);
-    UpdateResult();
+    UpdateResult(ctc_log_probs);
+    finish = IsEndpoint(ctc_log_probs) || finish;
```

Comment:
We should add more states tag to show what state we are.
1. normal decoding state
2. end of feature
3. endpoint